backend/tests/integration/schema_lifecycle/test_schema_attribute_remove_add.py

```diff
@@ -0,0 +1,210 @@
+        persons = await registry.manager.query(db=db, schema=PERSON_KIND)
+        assert len(persons) == 1
+
+    async def test_step02_check_attr_add_rename(
+        self, db: InfrahubDatabase, client: InfrahubClient, initial_dataset, schema_step02
+    ):
+        success, response = await client.schema.check(schemas=[schema_step02])
+        assert success
+        assert response == {
+            "diff": {
+                "added": {},
+                "changed": {
+                    "TestingPerson": {
+                        "added": {},
+                        "changed": {
+                            "attributes": {
+                                "added": {},
+                                "changed": {},
+                                "removed": {"height": None},
+                            },
+                        },
+                        "removed": {},
+                    },
+                },
+                "removed": {},
+            },
+        }
+
+    async def test_step02_load(self, db: InfrahubDatabase, client: InfrahubClient, initial_dataset, schema_step02):
+        response = await client.schema.load(schemas=[schema_step02])
+        assert not response.errors
+
+        # Ensure that we can query the nodes with the new schema in BRANCH1
+        persons = await registry.manager.query(
+            db=db,
+            schema=PERSON_KIND,
+            filters={"firstname__value": "John"},  # , branch=self.branch1
+        )
+        assert len(persons) == 1
+        john = persons[0]
+        assert john.firstname.value == "John"  # type: ignore[attr-defined]
+        assert not hasattr(john, "height")
+
+    async def test_step03_check(self, db: InfrahubDatabase, client: InfrahubClient, initial_dataset, schema_step03):
+        success, response = await client.schema.check(schemas=[schema_step03])
+        assert response == {
+            "diff": {
+                "added": {},
+                "changed": {
+                    "TestingPerson": {
+                        "added": {},
+                        "changed": {
+                            "attributes": {"added": {"height": None}, "changed": {}, "removed": {}},
+                        },
+                        "removed": {},
+                    },
+                },
+                "removed": {},
+            },
+        }
+        assert success
+
+    async def test_step03_load(self, db: InfrahubDatabase, client: InfrahubClient, initial_dataset, schema_step03):
+        response = await client.schema.load(schemas=[schema_step03])
+        assert not response.errors
+
+        # Modify the value for Height in the database
+        persons = await registry.manager.query(
+            db=db,
+            schema=PERSON_KIND,
+            filters={"firstname__value": "John"},
+        )
+        assert len(persons) == 1
+        john = persons[0]
+        john.height.value = 200
```

Comment:
maybe add an assert that the value is `None` before updating it to check that John's initial height was deleted